### PR TITLE
modify controller for view

### PR DESCRIPTION
**Pull request by Haadiiii**
## BLOG-APP View :arrow_double_down: 

In this Activity: 

> Controllers

- Modify controller for:
   - Users to select all users and get `author_id` for specific user.
   - Posts to select all Posts and get `post_id` for specific users.
 
> View

- Modify index and show file and add plan HTML .

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,44 @@
...
+    <div class="card mb-3 border-danger" style="width: 680px;">
+        <h2 class="card-title bg-info">Posts</h2>
+
+            <% @user.posts.each do |post| %>
+                    <%= link_to post.title, user_post_path(@user, post) %>
+        <ul class="card-body">
+                <li class="card-text">
+                    <%= post.text %>
+                </li>
+        </ul>
```
Comment (codepantha) [inline app/views/users/show.html.erb:32]: - [ ] It is expected that in the users `show` page, the `most_recent_posts` method that you have already defined in the `User` model is used to display the 3 most recent posts. Right now, you're displaying all the user's posts. Please try to fix this 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @Haadiiii 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Created views for your app
✔️ Used GitFlow
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (Haadiiii) [reply app/views/users/show.html.erb]: done :+1:

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Haadiiii,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**